### PR TITLE
Add TDP 48 & 49 - Update Headers

### PR DESCRIPTION
**Pull request by mzero2nine**
Hi @dimsome 

Can you please review this? I tried to structure it out as good as possible, but good to get a 2nd set of eyes over this, thank you! 😊

### CODE REVIEW

File: TDP/tdp-48.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+### Builder subDAO Representative
+For communications with the mStableDAO, the nominated representative for the Builder subDAO will be the Builder subDAO’s Head of Operations. The representative will be accountable for ensuring that communication and transparency reporting requirements are met by the Builder subDAO. If the representative changes, the mStableDAO Cat Herder will be notified.
+
+### Internal Decision-making
+The subDAO signer group will have discretionary power to disburse subDAO funds, provided that the funding use is in line with the proposed funding categories and within the total budgeted amount for the period, including any buffer amount.
+
+Any spending for uses outside the proposed categories would require an additional governance process.
+
+### Proposed budget
+### mUSD
```
Comment (dimsome) [inline TDP/tdp-48.md:62]: two headlines here, let's make it one

File: TDP/tdp-48.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+* Make proposals to governance to implement changes impacting mStable’s smart contracts
+* Develop and maintain the mStable front end application built on top of the mStable protocol
+* Work closely with other subDAOs to promote consistency and alignment across the mStable ecosystem, including the provision of design services and developer support to the Ecosystem subDAO
+
+## Specification
+
+### SubDAO Structure
+It is proposed that a new Gnosis Safe be created on Ethereum Mainnet, with the option to deploy additional safes with the same signer structure on Polygon or any other chains in the future to meet the operational needs of the subDAO. The proposed signer structure is a 2/3 multisig, with the signer group consisting of 3 current mStable core team members who will contribute to the Builder subDAO. The proposed signer addresses are as follows:
+
+* 0x89901e7d025df331e9eC84120D3f81603e50364e (Core Team Member)
```
Comment (dimsome) [inline TDP/tdp-48.md:47]: put it in back ticks `93939393`

File: TDP/tdp-48.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+title: Funding Request - mStable Builder subDAO (Sep-Dec 2022)
+status: Proposed
+author: Cam Soulsby (@camsoulsby),  mZeroNine (@mzero2nine),  0xloth (@LDeroccha),  
+discussions-to: https://forum.mstable.org/t/funding-request-mstable-builder-subdao-sep-dec-2022/924
+created: 2022-08-26
+---
+
+## Simple Summary
+
+This proposal requests 4 months of operational funding for the proposed mStable Builder subDAO, originally defined in [TDP 44](https://mips.mstable.org/TDP/tdp-44.html). The proposal will:
```
Comment (dimsome) [inline TDP/tdp-48.md:12]: relative link

File: TDP/tdp-49.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+* Planning of Community Events (Trivia, Game Nights, etc.)
+* Operation of the Dework Bounty Board
+* Researching new ways to better the mStable community i.e. the integration of platforms such as Dework
+* Managing and creating content for mStable
+
+A full itemized list of funding initiatives can be found in the “Amount Requested” portion of this Proposal. Funding received will be utilized to optimally run the subDAO to carry out its initiatives and meet its objectives.
+
+## Motivation
+
+The Ecosystem subDAO Funding Request is created in order to resolve the merger between the Community subDAO and the GrantsDAO ratified in [TDP 46](https://mips.mstable.org/TDP/tdp-46.html)
```
Comment (dimsome) [inline TDP/tdp-49.md:32]: relative link here

File: TDP/tdp-49.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+* trustindistrust - Community subDAO operations i.e. championing Dework
+* Fungus - Community and subDAO manager/lead. Responsible for efficient operation of the subDAO and the obtaining of its objectives
+
+### Community Manager Role & Responsibilities
+To ensure a smooth experience for both the community and mStable’s core contributors when interacting with the Ecosystem subDAO, Fungus’ role will be expanded in order to facilitate and allow for an increased set of responsibilities.
+
+In addition to the responsibilities below, the Community Manager will take a leadership role in the Ecosystem subDAO by guiding and directing priorities and providing oversight of the subDAO’s operations. This includes acting as the lead signer of the subDAO and main point of contact for any queries and other liaisons within the wider mStable ecosystem relating to the Ecosystem subDAO.
+
+Further information on lead signer responsibilities are outlined in [TDP-47](https://forum.mstable.org/t/tdp-47-mstable-restructure-compensation-model-subdao-funding-request-framework/897).
```
Comment (dimsome) [inline TDP/tdp-49.md:48]: relative link

File: TDP/tdp-49.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+
+Other responsibilities include but are not limited to:
+
+1.) Schedule and organize AMAs, Governance Calls, Community Calls, Metanaut Spotlight Calls, Friday Hangouts and other social events in the Official [mStable Discord](https://discord.com/invite/mstable).
```
Comment (dimsome) [inline TDP/tdp-49.md:52]: hopefully we keep this vanity link :D

File: TDP/tdp-49.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+
+Any surplus of mUSD or MTA at the end of the funding period, after accounting for any outstanding accounts payable, will be returned to the TreausuryDAO.
+
+### Terms of Request: Milestones, KPI’s
+**Timeline**: 9/22 - 12/22. This funding request will carry the Ecosystem subDAO through the remainder of 2022. A monthly report will be created to track spending on initiatives, and list upcoming ventures.
+
+* Successful utilization of Dework, defined through the regular successful posting, quality assurance, task completion, and payout of various bounties where previously defined deliverables that add value to the mStable ecosystem are met
+* Hosting of regular community events: Community Calls, AMA’s, Game Nights, Trivia where community members are rewarded through their participation via one or more of the following: POAP’s, NFT’s, MTA rewards, special discord roles, recognition in official announcements, and so on.
+* Miscellaneous community engagement activities such as the music competitions, raffles, and so on
+* Agreed upon report will be posted to the forum on a monthly basis per the requirements of [TDP-47](https://mips.mstable.org/TDP/tdp-47.html)
```
Comment (dimsome) [inline TDP/tdp-49.md:106]: relative link

File: TDP/tdp-48.md [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+Where salaries are denominated in Euro, an exchange rate of 0.96 USD/EUR has been used for conversion to allow small amount of room for movement in the rate.
+
+All unallocated budget after 4 months will be returned to the TreasuryDAO or, at the discretion of the TreasuryDAO, may be subtracted from any subsequent approved funding request. Note that any calculation of unallocated budget should take into account any outstanding accounts payable to ensure that the subDAO can meet future obligations.
+
+To ensure that the subDAO is not incentivized to overspend and use the buffer amount, the TreasuryDAO should hold the Builder subDAO accountable to the original budget and consider use of the buffer as a negative indicator of performance.
+
+### MTA
+In addition to the proposed funding in USD stablecoins, a total amount of 1,175,000 MTA is requested to allow the Builder subDAO to handle the MTA allocations for current team members as per their current agreements with the mStableDAO.
+
+![MTA Breakdown](../assets/TDP-48/mta-breakdown.png)
```
Comment (dimsome) [inline TDP/tdp-48.md:88]: this looks like a table, you can add tables in markdown

**Review by dimsome — CHANGES_REQUESTED** [review]
a few adjustments plz

File: TDP/tdp-48.md (hunk 1 above)
Comment (mzero2nine) [reply TDP/tdp-48.md]: Fixed

File: TDP/tdp-48.md (hunk 2 above)
Comment (mzero2nine) [reply TDP/tdp-48.md]: Fixed

File: TDP/tdp-48.md (hunk 3 above)
Comment (mzero2nine) [reply TDP/tdp-48.md]: Fixed

File: TDP/tdp-49.md (hunk 4 above)
Comment (mzero2nine) [reply TDP/tdp-49.md]: Fixed

File: TDP/tdp-49.md (hunk 5 above)
Comment (mzero2nine) [reply TDP/tdp-49.md]: Fixed

File: TDP/tdp-49.md (hunk 6 above)
Comment (mzero2nine) [reply TDP/tdp-49.md:52]: LVL 3 forever!

File: TDP/tdp-49.md (hunk 7 above)
Comment (mzero2nine) [reply TDP/tdp-49.md]: Fixed

File: TDP/tdp-48.md (hunk 8 above)
Comment (mzero2nine) [reply TDP/tdp-48.md]: Fixed

**Comment by dimsome** [comment]
The 2 png files can be deleted now, right?

**Comment by mzero2nine** [comment]
> The 2 png files can be deleted now, right?

Yes of course, I wanted to do that, but forgot in the heat of things (I always remember why I'm not a dev!)

**Comment by dimsome** [comment]
Could you please do that and commit the change? I cannot delete files on your branch

**Review by dimsome — APPROVED** [review]
Thank you!

**Comment by mzero2nine** [comment]
> Could you please do that and commit the change? I cannot delete files on your branch

Learned a lot today from editing and doing things here on GH! Sorry if I cause smoothbrain pangs to you ser!

**Comment by dimsome** [comment]
No worries! Thanks

**Pull request merged**